Sources/SpeziQuestionnaire/QuestionnaireView.swift

```diff
@@ -75,7 +75,7 @@ public struct QuestionnaireView: View {
         questionnaireResponse: (@MainActor (QuestionnaireResponse) async -> Void)? = nil
     ) {
         self.questionnaire = questionnaire
-        self._isPresented = isPresented
+        self._isPresented = isPresented // TODO: this is not optional!
```

Comment:
There is still a TODO in here. Happy to discuss what we should do here in a few minutes.